src/Uno.UI.RemoteControl.Messaging/Uno.UI.RemoteControl.Messaging.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+
+		<PackageId>Uno.UI.RemoteControl.Messaging</PackageId>
+		<PackageId Condition="'$(UNO_UWP_BUILD)'=='false'">Uno.WinUI.RemoteControl.Messaging</PackageId>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<Compile Include="..\Uno.UI.RemoteControl.Server\IRemoteControlServer.cs" Link="IRemoteControlServer.cs" />
+		<Compile Include="..\Uno.UI.RemoteControl.Server\IServerProcessor.cs" Link="IServerProcessor.cs" />
+		<Compile Include="..\Uno.UI.RemoteControl\Messages\Frame.cs" Link="Messages\Frame" />
+		<Compile Include="..\Uno.UI.RemoteControl\Messages\IMessage.cs" Link="Messages\IMessage" />
```

Comment:
Are these only compiled in Uno.UI.RemoteControl.Messaging project? In that case they should be moved to this directory and avoid `<Compile [Include|Remove]`

Reply:
I think it should be safe to move the files, but let's see if @jeromelaban can confirm

Reply:
moved by Jerome